pages/bookmarks.jsx

```diff
@@ -37,7 +37,7 @@ export default React.createClass({
     return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
   },
   getAnonymousContent() {
-    return (<p>Are you Mozilla Staff? <button className="btn btn-link inline-link" onClick={this.handleSignInBtnClick}>Sign in</button>.</p>);
+    return (<p>Are you Mozilla Staff? <a href="javascript:void(0)" onClick={this.handleSignInBtnClick}>Sign in</a>.</p>);
```

Comment:
Hey sorry @Sudip7 . Based on our discussion above it turns out the current code is good as is - we don't have to change it to `<a>`. As a result I'm gonna close this PR as well as the original ticket that I filed https://github.com/mozilla/network-pulse/issues/365 . I'll try to find an intro ticket and make sure it *is valid* before asking someone to tackle it. Sorry for the confusion. 😁